admin/services/psychometrician-report.service.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+const psCachedReportDataService = require('./data-access/ps-report-cache.data.service')
+const completedCheckDataService = require('./data-access/completed-check.data.service')
+const checkDataService = require('./data-access/check.data.service')
+const dateService = require('./date.service')
+
+const psychometricianReportService = {}
+
+psychometricianReportService.generateReport = async function () {
+}
+
+psychometricianReportService.batchProduceCacheData = async function (batchIds) {
+  if (!batchIds) {
+    throw new Error('Missing argument: batchIds')
+  }
+
+  if (!(Array.isArray(batchIds) && batchIds.length)) {
+    throw new Error('Invalid arg: batchIds')
+  }
+
+  const completedChecks = await completedCheckDataService.find({_id: {'$in': batchIds}})
+  await this.populateWithCheck(completedChecks)
+
+  for (let check of completedChecks) {
+    await this.produceCacheData(check)
+  }
+}
+
+psychometricianReportService.produceCacheData = async function (completedCheck) {
+  if (!completedCheck) {
+    throw new Error('Missing argument: completedCheck')
+  }
+
+  if (!(typeof completedCheck === 'object' && completedCheck._id)) {
+    throw new Error('Invalid argument: completedCheck')
+  }
+
+  const psData = {
+    'AttemptId': completedCheck.check.checkCode,
+    'DOB': dateService.formatUKDate(completedCheck.check.pupilId.dob),
+    // 'estab': answer.school.estabCode,
```

Comment:
Are you going to uncomment these later?
If not you might as well remove them completely.